packages/rule-components/src/RuleTable/RuleTable.js

```diff
@@ -177,9 +179,31 @@ class RuleTable extends Component {
                                 props: { colSpan: columns.length + Boolean(actions) }
                             }]
                         })) :
-                        flatten(createRows(data, columns, expanded, detail)).filter(Boolean)
+                        data.length > 0 ?
+                            flatten(createRows(data, columns, expanded, detail)).filter(Boolean) :
+                            [{
+                                heightAuto: true,
+                                cells: [
+                                    {
+                                        props: { colSpan: 8 },
+                                        title: (
+                                            <Bullseye>
+                                                <EmptyState variant={EmptyStateVariant.small}>
+                                                    <EmptyStateIcon icon={SearchIcon} />
```

Comment:
Can you add `emptyIcon` property so anybody using this component can change it. You can assign `SearchIcon` as default prop.